### PR TITLE
Record Atrac library version

### PR DESCRIPTION
**Pull request by sum2012**
Gripshift Atrac library version 0x101 ,crc 3dd5e32f

### CODE REVIEW

**Comment by hrydgard** [comment]
This makes sense to have. Just some minor comments, I can also fix it up if you like.

File: Core/HLE/sceAtrac.cpp [hunk 1]
```diff
@@ -657,9 +657,12 @@ const int PSP_NUM_ATRAC_IDS = 6;
 static bool atracInited = true;
 static Atrac *atracIDs[PSP_NUM_ATRAC_IDS];
 static u32 atracIDTypes[PSP_NUM_ATRAC_IDS];
+static int atracLibVersion = 0;
+static u32 atracLibCrc = 0;
 
 void __AtracInit() {
 	atracInited = true;
+	INFO_LOG(ME, "AtracInit, atracLibVersion 0x%0x, atracLibcrc %x", atracLibVersion, atracLibCrc);
```
[the remark this answers is not in the document]
Comment (sum2012) [reply Core/HLE/sceAtrac.cpp]: The log moved to  _AtracSetData.
 initial state is defined 0 in https://github.com/hrydgard/ppsspp/pull/18392/files#diff-209847d78f1dab4ca702cda37ae91f0b9d34c3e4d9a4e45dd48ada8aefb7a3caR661

File: Core/HLE/sceAtrac.cpp [hunk 2]
```diff
@@ -697,6 +705,14 @@ void __AtracDoState(PointerWrap &p) {
 		}
 	}
 	DoArray(p, atracIDTypes, PSP_NUM_ATRAC_IDS);
+	if (s < 2) {
+		atracLibVersion = 0;
+		atracLibCrc = 0;
+	}
+	else {
+			Do(p, atracLibVersion);
+			Do(p, atracLibCrc);
```
[the remark this answers is not in the document]
Comment (sum2012) [reply Core/HLE/sceAtrac.cpp]: Fixed

File: Core/HLE/sceAtrac.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (hrydgard) [reply Core/HLE/sceAtrac.cpp]: Hm, that won't be spammy? maybe just log directly in __AtracLoadModule?

**Pull request merged**